components/inputs/docs/input-color.md

```diff
@@ -0,0 +1,60 @@
+# Color Inputs
+
+Color inputs allow users to select a color from a palette and perform color contrast analysis.
+
+> **Note**: Color inputs rely on functionality only available in the MVC framework, and can therefore only be used inside a MVC page. This restriction may go away in the future as more functionality is built into the color input web component itself.
+
+<!-- docs: demo -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-color.js';
+</script>
+<d2l-input-color type="foreground" value="#ff0000"></d2l-input-color>
+```
+
+## Color Input [d2l-input-color]
+
+The `<d2l-input-color>` will open a dialog to allow the user to select a color from a palette.
+
+<!-- docs: demo live name:d2l-input-color -->
```

Comment:
I'm tempted to not add this component to the Daylight Site just yet seeing as it's so reliant on the MVC framework. Curious what others think.

Reply:
I am also on the fence, but I think I lean towards including it so that designers/devs know what we have and where it stands.

Reply:
Would designers reading this page understand what the MVC framework is and whether they are able to use the component? I do agree that I think it would probably be better to have the documentation there than not, but I can see that causing confusion.

Reply:
They might have a general understanding, but yeah it's mostly a dev-focused message. The important thing I think is that it can be used everywhere except FRAs (and we could technically make it work there if we really wanted to).